### ISSUE TITLE
Result is out of order when using http stream mode

### ISSUE THREAD

**Issue by liu21yd** [triton-inference-server/tensorrtllm_backend#614]
1. I built the image from source , using the TensorRT-LLM and tensorrtllm_backend main brand at September 25th。
2. I built qwen2.5 14B and start the triton server successfully.

I get http result like this:
![roNhU7fK79](https://github.com/user-attachments/assets/46aa5e8f-3ba0-43c0-9d4f-332471813802)
but I get a out of order result by using generator_stream url:
![img_v3_02fh_0b3f5b1c-569a-4476-857a-7a063fb8ecfg](https://github.com/user-attachments/assets/a3b92d5b-6786-4722-b979-a5aec98df270)

Who can help me?

**Comment by intellimouseftw** [comment]
+1 I am facing the same issue. Are you using >1 for `postprocessing_instance_count` in your config.pbtxt?

**Comment by matichon-vultureprime** [comment]
+1 I am using version 0.13.0 rel. 
The issue occurs when I set `postprocessing_instance_count` > 1. 
Even if I set `postprocessing_instance_count` > 1 and match it with `preprocessing_instance_count`, it still doesn't resolve the issue.

In version 0.12.0 rel, it worked well.

**Comment by intellimouseftw** [comment]
I just tried version 0.12.0, it worked well on there with `postprocessing_instance_count` > 1. 

Seems like a bug introduced in 0.13.0

**Comment by liu21yd** [comment]
> +1 I am facing the same issue. Are you using >1 for `postprocessing_instance_count` in your config.pbtxt?

Yes. When I use the v0.11.0.dev2024051400 version's config, it works. It seems that some new features caused this bug.
